project/static/index.html

```diff
@@ -13,8 +13,10 @@
     <!-- scripts -->
     <script src="//code.jquery.com/jquery-2.1.4.min.js" type="text/javascript"></script>
     <script src="//netdna.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" type="text/javascript"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.9/angular.min.js" type="text/javascript"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.9/angular-route.min.js" type="text/javascript"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.10/angular.min.js" type="text/javascript"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.10/angular-route.min.js" type="text/javascript"></script>
     <script src="static/app.js" type="text/javascript"></script>
+    <script src="static/services.js" type="text/javascript"></script>
+    <script src="static/controllers.js" type="text/javascript"></script>
   </body>
 </html>
```

Comment:
Please put a blank line at the end of file.

Ensure that for every file.

Read this: http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file